src/main/webapp/script.js

```diff
@@ -211,3 +214,35 @@ for (let button of filterButtons) {
       }
     });
 }
+
+let infowindow;
+function updateRestaurantInfo(restaurant, marker) {
+  if (!infowindow) {
+    infowindow = new google.maps.InfoWindow();
+    infowindow.setZIndex(500); // Random high number so it shows on top
+  }
+  let { geometry: {location}, placeId, name, photos, rating } = restaurant;
+
+  let params = {
+    api: 1,
```

Comment:
What does 1 for api mean?

Reply:
It's just a required parameter, no real meaning behind it. 

"The parameter api=1 identifies the version of Maps URLs this URL is intended for. This parameter is required in every request. The only valid value is 1."

https://developers.google.com/maps/documentation/urls/guide